RELEASES.md

```diff
@@ -4,6 +4,9 @@
 - Bump golang version to v1.23.6
 - Bump golangci-lint to v1.63 and add linters
 
+- Added `eth_suggestPriceOptions` API to suggest gas prices (slow, normal, fast) based on the current network conditions
+- Added `"po-slow-fee-percentage"`, `"po-fast-fee-percentage"`, `"po-max-base-fee"`, and `"po-max-tip"` config flags to configure the new `eth_suggestPriceOptions` API
```

Comment:
is `po` standard to mean price options? Perhaps i.e. `price-option-slow-fee-percentage` would be clearer?

Reply:
Yes I think it needs a prefix to specify this is applied to `priceOption` API. I was skeptical having such a long name, but surely can do.